metashare/api/tests/jobs.py

```diff
@@ -110,11 +110,11 @@ def test_get_unsaved_changes(scratch_org_factory):
         get_unsaved_changes(scratch_org=scratch_org)
         scratch_org.refresh_from_db()
 
-        assert scratch_org.unsaved_changes
-        assert scratch_org.latest_revision_numbers == {
-            "TypeOne": {"NameOne": 13},
-            "TypeTwo": {"NameTwo": 10},
+        assert scratch_org.unsaved_changes == {
+            "TypeOne": ["NameOne"],
+            "TypeTwo": ["NameTwo"],
         }
+        assert scratch_org.latest_revision_numbers == {"TypeOne:NameOne": 10}
```

Comment:
We're not using this format anymore; if you look at what `get_latest_revision_numbers` returns, it's `Dict<str, Dict<str, int>>`.

Reply:
I'm fighting a fire elsewhere; would you mind fixing? I just intended to assert that it's the same as what it was initialized to at the beginning of the test.

Reply:
Yeah, I'm running this through functional tests locally to make sure I don't miss anything, and I'll push some fixes.